README.md

```diff
@@ -17,4 +17,4 @@ RapidSmith2 (and previously RapidSmith) have been used in a variety of settings
 
 For the full documentation, see the [TechReport](doc/TechReport.pdf).
 
-**NOTE**: The current release of RS2 is an alpha release. Significant changes to the code are still being made, and so expect frequent code updates. RS2 and its functionality is still being tested, so please report any bugs you find. Also, only designs that can be fully flattened to Xilinx leaf primitive cells is currently supported. A solution to support macro primitives is currently being implemented. 
\ No newline at end of file
+**NOTE**: The current release of RS2 is an alpha release. Significant changes to the code are still being made, and so expect frequent code updates. RS2 and its functionality is still being tested, so please report any bugs you find. Also, only designs that can be fully flattened to Xilinx leaf primitive cells is currently supported. A solution to support macro primitives is currently being implemented.
```

Comment:
I changed this file in VIM, but it still removed that last windows line ending. Strange. (changing other files doesn't do this)